### DISCUSSION TITLE
User account creations are slow

### DISCUSSION THREAD

**Discussion by norrcomm** [nilsteampassnet/TeamPass#3683] [Q&A]
Now that TeamPass is set up, I am trying to set up users.

It has been more than 24 hours and the user account is not ready, they have an animated hourglass icon beside them.

Is this normal?  [CLICK HERE](https://prnt.sc/LyzlB6qYmMqP)

**Comment by nilsteampassnet** [comment]
@norrcomm You must have an error in the setup of task cron.
Please refer to https://documentation.teampass.net/#/manage/tasks

**Comment by Mwendlin** [comment]
are there any instructions for setting up the cron job on windows server?

**Reply by nilsteampassnet** [reply]
You can find on Google.
An example : https://linuxhint.com/set-up-cron-job-windows/

WOuld be nice to come back when you did it on your Windows server, thanks

**Reply by Arash9703** [reply]
Need your idea : https://github.com/nilsteampassnet/TeamPass/discussions/3792#discussioncomment-7777423
Thanks
